Permanent/Modules/EditMetadata/Views/MetadataEditView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  MetadataEditView.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 25.06.2023.
+//
+
+import SwiftUI
+
+struct MetadataEditView: View {
+    @State private var fileName: String = ""
```

Comment:
Do you use this variables? If not, please remove them as they will probably be in the view model